site.yml

```diff
@@ -18,47 +18,34 @@ site:
             $ref: gh_pages
         data:
             nav:
-                - name: Home
+                - title: Home
                   url: /
```

Comment:
I think this `url` can be converted into the `href` for the built-in structure, that you indicate as missing attribute in the commit message body.

Reply:
Actually, I tried that, but it didn't resolve the failure. I think I should investigate further, but think this is a nice increment and enough to close the simplification issue